ultravox/tools/ds_tool/ds_tool.py

```diff
@@ -175,6 +176,97 @@ def _map_sample(self, sample, exclude_fields):
         return sample
 
 
+@dataclasses.dataclass
+class AudioExtensionTask:
+    audio_column_name: str = simple_parsing.field(default="audio", alias="-a")
+    asr_column_name: str = simple_parsing.field(default="sentence", alias="-A")
+    translation_column_name: str = simple_parsing.field(
```

Comment:
some datasets (eg librispeech) won't have this. Should this be optional?

Reply:
Yes, good point. We should be able to process transcription only datasets as well.